CoffeeChat/Controller/EditTimeAvailabilityViewController.swift

```diff
@@ -0,0 +1,320 @@
+        let eveningSection = TimeSection(type: .morning, items: eveningItems)
+        
+        timeSections = [morningSection, afternoonSection, eveningSection]
+    }
+    
+    private func setupConstraints() {
+        let dayCollectionViewWidth = daysAbbrev.count * 45
+
+        scheduleTimeLabel.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(20)
+            make.centerX.equalTo(view.snp.centerX)
+        }
+        
+        dayCollectionView.snp.makeConstraints { make in
+            make.top.equalTo(scheduleTimeLabel.snp.bottom).offset(20)
+            make.centerX.equalTo(view.snp.centerX)
+            make.height.equalTo(50)
+            make.width.equalTo(dayCollectionViewWidth)
+        }
+        
+        dayLabel.snp.makeConstraints { make in
+            make.top.equalTo(dayCollectionView.snp.bottom).offset(10)
+            make.centerX.equalTo(view.snp.centerX)
+        }
+        
+        timeCollectionView.snp.makeConstraints { make in
+            let timeCollectionViewWidth = timeSections.count * 105
+            let timeCollectionViewHeight = 400
+            make.top.equalTo(dayLabel.snp.bottom).offset(20)
+            make.height.equalTo(timeCollectionViewHeight)
+            make.width.equalTo(timeCollectionViewWidth)
+            make.centerX.equalTo(view.snp.centerX)
+        }
+        
+    }
+    
+}
+
+extension EditTimeAvailabilityViewController: UICollectionViewDataSource {
+    
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        if collectionView == dayCollectionView {
+            return 1
+        } else if collectionView == timeCollectionView {
+            return timeSections.count
+        }
+        return 2
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        if collectionView == dayCollectionView {
+            return daysAbbrev.count
+        }
+        return timeSections[section].items.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        if collectionView == dayCollectionView {
+            guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: dayCellReuseId, for: indexPath) as? SchedulingDayCollectionViewCell else { return UICollectionViewCell() }
+            let day = daysAbbrev[indexPath.item]
+            cell.configure(for: day)
+            if let day = daysDict[day] {
+                let isAvailable = availabilities[day] != nil
+                cell.updateBackgroundColor(isAvailable: isAvailable)
+            }
+            if day == selectedDay {
+                dayCollectionView.selectItem(at: indexPath, animated: false, scrollPosition: .left)
+                cell.isSelected = true
+            }
+            return cell
+        }
```

Comment:
I'd put a newline between the end bracket and `let section...`, to make is visually clear that above we are handling the case for `dayCollectionView` and below we are handling the other case